cmd/api/src/api/registration/v2.go

```diff
@@ -114,7 +114,7 @@ func NewV2API(cfg config.Configuration, resources v2.Resources, routerInst *rout
 		routerInst.GET("/api/version", v2.GetVersion).RequireAuth(),
 
 		// API Spec
-		routerInst.PathPrefix("/api/v2/swagger", v2.SwaggerHandler()),
+		routerInst.PathPrefix("/api/v2/swagger", http.HandlerFunc(openapi.HttpHandler)),
```

Comment:
Please pay close attention to this change as it does introduce slightly strange behavior at the tradeoff of being much simpler and straightforward while maintaining backwards compat (aka `/api/v2/swagger/<literallyanything>` will respond with a 200 and the json doc)

Reply:
I think this is fine, so long as there weren't any additional weird functionality that the swagger handler provided on this prefix. That would mean it's backwards compatible which is probably the only thing that truly matters here